Assets/MRTK/Core/Definitions/InputSystem/PointerOption.cs

```diff
@@ -16,11 +16,12 @@ public struct PointerOption
         /// <summary>
         /// Constructor.
         /// </summary>
-        public PointerOption(SupportedControllerType controllerType, Handedness handedness, GameObject pointerPrefab)
+        public PointerOption(SupportedControllerType controllerType, Handedness handedness, GameObject pointerPrefab, LayerMask[] prioritizedLayerMasks)
```

Comment:
can we pass a default value for this new parameter to prevent it from being as breaking?

Reply:
Had to pass in null as the default parameter, but added a sensible auto-populate field within the struct declaration